titus-ext/jooqflyway/src/main/java/com/netflix/titus/ext/jooqflyway/jobactivity/JooqJobActivityConsumerStore.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.ext.jooqflyway.jobactivity;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.netflix.titus.api.jobmanager.model.job.Job;
+import com.netflix.titus.api.jobmanager.model.job.Task;
+import com.netflix.titus.runtime.endpoint.common.LogStorageInfo;
+import com.netflix.titus.supplementary.jobactivity.store.JobActivityConsumerStore;
+import org.jooq.DSLContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Qualifier;
+import reactor.core.publisher.Mono;
+
+
+@Singleton
+public class JooqJobActivityConsumerStore implements JobActivityConsumerStore {
+
+    private static final Logger logger = LoggerFactory.getLogger(JobActivityConsumerStore.class);
+    private final LogStorageInfo<Task> logStorageInfo;
+    private final DSLContext jobActivityDslContext;
+    private final DSLContext producerDSLContext;
+
+
+    @Inject
+    public JooqJobActivityConsumerStore(@Qualifier("jobActivityDSLContext") DSLContext jobActivityDSLContext,
+                                        @Qualifier("producerDSLContext") DSLContext producerDSLContext,
+                                        LogStorageInfo<Task> logStorageInfo) {
```

Comment:
We should not depend on this class here. Log data are already attached to protobuf Task object.